server.js

```diff
@@ -3266,23 +3270,38 @@ cache(function(data, match, sendBadge, request) {
       var data = JSON.parse(buffer);
       var modifier = '';
       var issues;
-      if (isPR) {
-        issues = data.total_count;
+      if (isState && issuesApi) {
+        var rightSide = isRaw ? data.state : data.title;
+        rightSide = rightSide.length > 25 ? rightSide.substr(0,21)+'...' : rightSide
```

Comment:
Maybe pull these out as constants (eg. `const MAX_TITLE_LENGTH = 25` or something)

I feel like this should be a separate utility function, like
```
function truncateTitle(str) {
  return str.length > MAX_TITLE_LENGTH
    ? str.substr(0, MAX_TITLE_LENGTH - 4) + '...'
    : str;
}
```